CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 ## Changelog
 
+### 0.21.5
+
+**Commit Delta**: [Change from 0.21.4 release](https://github.com/plus3it/watchmaker/compare/0.21.4...0.21.5)
+
+**Released**: 2021.02.25
+
+**Summary**:
+
+*   ash-linux-formula
+    - Replace watch with listen to address service restarts
+    - Make state RHEL-07-040560 more resilient when the yum group info is missing
+*   scap-formula
+    - Updates SCAP content from DISA (as of February 2021) and OpenSCAP (v0.1.54)
+*   watchmaker-salt-content
+    - Bump salt version to 2019.2.8
+*   Bump readme-renderer from 28.0 to 29.0
```

Comment:
We can remove this. We don't generally list out changes that are not user-facing. This one is only relevant to the CI/packaging processes...